File: src/main/java/com/matag/game/MatagGameConfiguration.java
package com.matag.game;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

import com.matag.cards.CardsConfiguration;
import com.matag.game.cardinstance.CardInstanceConfiguration;

@Configuration
@Import({CardsConfiguration.class, CardInstanceConfiguration.class})
public class MatagGameConfiguration {

}

File: src/main/kotlin/com/matag/game/MatagGameConfiguration.kt
package com.matag.game

import com.matag.cards.CardsConfiguration
import com.matag.game.cardinstance.CardInstanceConfiguration
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import

@Configuration
@Import(CardsConfiguration::class, CardInstanceConfiguration::class, MatagGameWebSocketConfiguration::class)
class MatagGameConfiguration 
